AndroidLibrary/Sanku Yogesh/LibUsage/app/src/main/java/android/example/libusage/MainActivity.java

```diff
@@ -0,0 +1,49 @@
+package android.example.libusage;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.lifecycle.Observer;
+import androidx.lifecycle.ViewModelProvider;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.example.libusage.model.RecyclerList;
+import android.os.Bundle;
+import android.widget.Toast;
+
+
+public class MainActivity extends AppCompatActivity {
+
+    RecyclerViewAdapter recyclerViewAdapter;
+    private MainActivityViewModel viewModel;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        initRecyclerView();
+        getData();
+    }
+
+    private void initRecyclerView(){
+        RecyclerView recyclerView = findViewById(R.id.recyclerView);
+        recyclerView.setLayoutManager(new LinearLayoutManager(this));
+        recyclerViewAdapter = new RecyclerViewAdapter();
+        recyclerView.setAdapter(recyclerViewAdapter);
+    }
+
+    private void getData(){
+        viewModel = new ViewModelProvider(this).get(MainActivityViewModel.class);
+        viewModel.getRecyclerListObserver().observe(this, new Observer<RecyclerList>() {
```

Comment:
Anonymous new Observer<RecyclerList>() should be replaced with lambda + Clean the code + Proper access modifiers should be there. 
Some fields should be final. 
Nullability needs to handle well.